Sources/SignHereLibrary/Services/iTunesConnectService.swift

```diff
@@ -373,7 +376,9 @@ internal class iTunesConnectServiceImp: iTunesConnectService {
         request.setValue(Constants.applicationJSONHeaderValue, forHTTPHeaderField: Constants.contentTypeHeaderName)
         request.setValue("Bearer \(jsonWebToken)", forHTTPHeaderField: "Authorization")
         request.httpMethod = "POST"
-        let profileName: String = "\(certificateId)_\(profileType)_\(clock.now().timeIntervalSince1970)"
+        // let profileName: String = "\(certificateId)_\(profileType)_\(clock.now().timeIntervalSince1970)"
+        let profileName: String = "API Created Profile: \(bundleIdentifier)"
```

Comment:
Making this more well-known is probably the right call. I believe Tinder uses this to create tool to create profiles to a well-known _path_, and references them directly in the `profiles = //some:profile.mobileprovision`

We use `local_provisioning_profile`, where yes the name is well-known. It might be worth passing this in via flag, but I think where we ultimately land in managing this chunk of a library will affect if that's the right choice.